anomalib/models/dfm/torch_model.py

```diff
@@ -129,11 +134,19 @@ def score(self, features: Tensor) -> Tensor:
             score = self.gaussian_model.score_samples(feats_projected)
         elif self.score_type == "fre":
```

Comment:
If I undertand correctly, segmentation is only supported when score_type is "fre"? Maybe you could mention this in the docstring of the method, and also in `anomalib/models/dfm/README.md`